routers/web/repo/release.go

```diff
@@ -200,6 +206,50 @@ func releasesOrTags(ctx *context.Context, isTagList bool) {
 	ctx.HTML(http.StatusOK, tplReleases)
 }
 
+// tags render the tags list if repo's release is disabled.
+func tags(ctx *context.Context) {
+	ctx.Data["PageIsTagList"] = true
```

Comment:
Maybe `Title` is missed here?
``` go
ctx.Data["Title"] = ctx.Tr("repo.release.tags")
```

Reply:
Thanks for your review. I'll fix it later.